### PR TITLE
Added this-week-in-os-18

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Created file,
- Added og-image

### CODE REVIEW

File: src/posts/2022-10-26-this-week-in-os-18.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+- [mainmatter/mainmatter-website-mailer]
+  [#3](https://github.com/mainmatter/mainmatter-website-mailer/pull/3) add CORS
+  headers ([@marcoow])
+- [mainmatter/mainmatter-website-mailer]
+  [#2](https://github.com/mainmatter/mainmatter-website-mailer/pull/2)
+  Auto-release from Actions ([@marcoow])
+- [mainmatter/mainmatter-website-mailer]
+  [#1](https://github.com/mainmatter/mainmatter-website-mailer/pull/1) Add CI
+  setup ([@marcoow])
+- [mansona/test-allow-failure]
```
Comment (mansona) [inline src/posts/2022-10-26-this-week-in-os-18.md]: this one can be ignored (I tend to prefix things in mansona with `test` if they are just test repos)

File: src/posts/2022-10-26-this-week-in-os-18.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  [#1](https://github.com/mansona/test-allow-failure/pull/1) add readme
+  ([@mansona])
+- [miragejs/miragejs] [#1064](https://github.com/miragejs/miragejs/pull/1064)
```
Comment (mansona) [inline src/posts/2022-10-26-this-week-in-os-18.md]: this can go in Javascript

**Review by mansona — COMMENTED** [review]
Just commenting on the two unknowns from me 👍

File: src/posts/2022-10-26-this-week-in-os-18.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+- [Turbo87/auto-dist-tag]
+  [#293](https://github.com/Turbo87/auto-dist-tag/pull/293) CI: Pin action
+  versions ([@Turbo87])
+- [Turbo87/auto-dist-tag]
+  [#292](https://github.com/Turbo87/auto-dist-tag/pull/292) renovate: Simplify
+  config by using shared config repo ([@Turbo87])
+
+## Unknown
+
+- [hakimel/reveal.js] [#3305](https://github.com/hakimel/reveal.js/pull/3305)
```
Comment (emmasofiah2o) [inline src/posts/2022-10-26-this-week-in-os-18.md]: @lolmaus can you help me classify this? You can find the different categories in the config file

File: src/posts/2022-10-26-this-week-in-os-18.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  Gulp livereload: include subfolders to watch for changes in html and md
+  ([@lolmaus])
+- [mainmatter/mainmatter-website-mailer]
```
Comment (emmasofiah2o) [inline src/posts/2022-10-26-this-week-in-os-18.md]: @marcoow should I include the PRs for this repository?

Comment (marcoow) [reply src/posts/2022-10-26-this-week-in-os-18.md]: yes you can

Comment (emmasofiah2o) [reply src/posts/2022-10-26-this-week-in-os-18.md]: What category should they be? Internal?

Comment (marcoow) [reply src/posts/2022-10-26-this-week-in-os-18.md]: yes

File: src/posts/2022-10-26-this-week-in-os-18.md (hunk 3 above)
Comment (lolmaus) [reply src/posts/2022-10-26-this-week-in-os-18.md]: It's `JavaScript`!

**Comment by emmasofiah2o** [comment]
@marcoow just to avoid confusion: this had already been reviewed so I am allowed to merge, but I have done a few changes since the last review. Does this mean I need to ask for a review again?

**Comment by marcoow** [comment]
So in general, review is not about being allowed to merge, it's about being certain that there aren't mistakes that the author of the PR hasn't recognized. The process isn't about authority or anything but it's driven by the understanding that nobody will produce perfect PRs at all times and we help each other in preventing accidental mistakes from being deployed to production.

If you only made some tiny changes, I guess the previous review is still fine – if you changed sth. that's non-trivial, it might make sense to request another review (see above – do the additional changes potentially include mistakes?)

**Comment by emmasofiah2o** [comment]
Ok gotcha, then I guess I can get a second review

**Comment by marcoow** [comment]
It's easier if you request from the entire engineering team – much better chance of someone being able to review quick.

**Review by mansona — APPROVED** [review]
looks good 🎉

**Review by BobrImperator — APPROVED** [review]

**Pull request merged**